### PR TITLE
doc: add source info into README

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Update the readme with the info where the users can find the catalog

**Motivation**
Closes; https://github.com/k8s-operatorhub/community-operators/issues/382

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 ## About this repository
 
 This repo is the canonical source for Kubernetes Operators that appear on [OperatorHub.io](https://operatorhub.io).
+The solutions merged on this repository are distributed via the [OLM][olm] index catalog [quay.io/operatorhubio/catalog][quay.io].
+Users can install [OLM][olm] in any Kubernetes or vendor such as Openshift to consume this content. [(more info)][catalog]
```
Comment (J0zi) [inline README.md:8]: I think, there is no need to describe manual OLM install, it is part of install output by hitting `Install`. Try it yourself https://operatorhub.io/operator/akka-cluster-operator

Comment (camilamacedo86) [reply README.md:8]: Any person can use the operators directly and without OLM. However, we built the catalogue/image to be used with OLM right? So, it seems to me that we need to describe that. So, the added info is specific to the image built and how it is used/useful for users.

**Comment by adambkaplan** [comment]
Has the image been moved to `catalog_sa`? The current `catalog` image in fact breaks when used with `opm index --add`.

See https://github.com/k8s-operatorhub/community-operators/commit/ae29062d83d412ff7550bcbe1ced36643ad9b470#diff-1b158839f8aa339e89bb50b1603e90ec68e0fd5565fe10a7cb32729fddf7070a

Side note - I could not find a pull request where this config change was discussed. @mvalarh is there a PR or issue that documents why this change was made?

**Comment by camilamacedo86** [comment]
Hi @adambkaplan,

Could you please raise an issue about the scenario of your comment https://github.com/k8s-operatorhub/community-operators/pull/452#issuecomment-985731014 that does not seem to have relation with the scope of this PR.

**Comment by adambkaplan** [comment]
@camilamacedo86 the issue regarding opm index is discussed in https://github.com/operator-framework/operator-registry/issues/870 (so not directly related to this community). Will continue discussing the issue there.

**Pull request merged**